### PR TITLE
feat: migrate existing data consortia roadmap to next.js (#2679)

### LINKED ISSUE #2679: [AnVIL Next] Migrate existing Data Consortia Roadmap to Next.js
#### Need
The existing [Data Consortia Roadmap](https://anvilproject.org/overview/data-consortia) page will be used as the new overview page for the consortia functionality.

#### Depends On
- https://github.com/clevercanary/anvil-portal/issues/4 must be completed before this task can be started.

#### Approach
* See [mocks](https://www.figma.com/file/JOUDXkKfgivRNFdBsWDPEH/AnVIL-Data-Explorer?type=design&node-id=2670-27422&t=jiibOC26nfX4FX1v-0).
* Add any missing typography to theme (e.g. list).
* Add CSER to list of current consortia and remove from planned consortia list, with the description: `The Clinical Sequencing Evidence-Generating Research (CSER) consortium is a national multi-site research program funded by the National Human Genome Research Institute (NHGRI), the National Cancer Institute (NCI), and the National Institute on Minority Health and Health Disparities (NIMHD).`
* Remove "Data Consortia Roadmap" from existing Portal navigation.
* Ingest over time chart is out of scope for this ticket.
* Link to consortia detail page is out of scope for this ticket.

### PR DESCRIPTION
**Pull request by frano-m**
Closes #2679.

### CODE REVIEW

File: next/scripts/update-ingestion-chart.js
Comment (frano-m) [inline next/scripts/update-ingestion-chart.js:1]: @hunterckx I apologise my `prettier` updated the whole `js` file. The only change should be to the constant `chartDataPath`.

File: package.json [hunk 1]
```diff
@@ -101,8 +99,7 @@
     "develop": "gatsby develop",
     "format": "prettier --write src/**/*.{js,ts,tsx}",
     "serve": "gatsby serve",
-    "test": "echo \"Write tests! -> https://gatsby.dev/unit-testing\"",
-    "update-ingestion-chart": "cd scripts && node update-ingestion-chart.js"
```
Comment (frano-m) [inline package.json:105]: Removed `update-ingestion-chart` script. Do we need one for `next.js`?

File: src/components/home/section-dataset-cloud/section-dataset-cloud.tsx [hunk 2]
```diff
@@ -9,21 +9,21 @@
 import React, { FC } from "react";
 
 // App dependencies
+import { DatasetCloudStaticQuery } from "../../../hooks/dataset-cloud-query";
```
Comment (frano-m) [inline src/components/home/section-dataset-cloud/section-dataset-cloud.tsx:12]: Again, `prettier` reordered the imports. The change here is the commenting out of the CTA button `Consortia Roadmap` until the project is live!

File: src/components/markdown/markdown.js [hunk 3]
```diff
@@ -11,32 +11,31 @@ import React, { useCallback, useEffect, useRef } from "react";
 import rehypeReact from "rehype-react";
 
 // App dependencies
-import Button from "../button/button";
+import ExternalLink from "../anchor-element/external-link/external-link";
 import ButtonLink from "../button-link/button-link";
-import DataIngestionChart from "../data-ingestion-chart/data-ingestion-chart";
```
Comment (frano-m) [inline src/components/markdown/markdown.js:16]: The only change here is removal of `DataIngestionChart`.

**Review by hunterckx — APPROVED** [review]

**Pull request merged**